### PR TITLE
Option to config python lib path from env var

### PR DESCRIPTION
**Pull request by villasv**
The simplest way to give the option to configure python library path from environment variables, targeting the issue https://github.com/tensorflow/serving/issues/216 about a scriptable complete configuration for tensorflow + serving.

### CODE REVIEW

**Comment by villasv** [comment]
@googlebot Done.

File: util/python/python_config.sh [hunk 1]
```diff
@@ -113,29 +113,33 @@ function setup_python {
...
+    if [ "$b" == "" ]; then
+      PYTHON_LIB_PATH="$(default_python_path "${python_lib_path[0]}")"
+      echo PYTHON_LIB_PATH
     else
-      echo -e "\n\nERROR: The path you have entered does not exist."
-      exit 1
+      PYTHON_LIB_PATH="$b"
     fi
   fi
+  if test -d "$PYTHON_LIB_PATH" -a -x "$PYTHON_LIB_PATH"; then
```
Comment (drpngx) [inline util/python/python_config.sh]: Why not an else branch? I guess this will only work with a single directory?

Comment (drpngx) [inline util/python/python_config.sh]: Ah, ok, that's fine then.

**Comment by drpngx** [comment]
Jenkins, test this please.

**Review by drpngx — APPROVED** [review]

**Pull request merged**